cumulus/polkadot-omni-node/lib/Cargo.toml

```diff
@@ -26,6 +27,9 @@ docify = { workspace = true }
 # Local
 jsonrpsee = { features = ["server"], workspace = true }
 parachains-common = { workspace = true, default-features = true }
+frame-metadata = { workspace = true, default-features = true, features = [
+	"decode",
```

Comment:
We also have an "unstable" feature for metadata: https://github.com/paritytech/frame-metadata/blob/961600149e44fe22e13e05c70c0da2cd1426201d/frame-metadata/Cargo.toml#L30-L31. dq: Would omni-nodes not expose the unstable metadata?

Reply:
I've discussed this with Iulian offline, and it seems this code is sufficient for the needs of omni-node atm (checking if a pallet exists). This is because either version of md (v14/15/16unstable) will contain the same information regarding collected pallets